<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graph</class>
 <widget class="QDialog" name="graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>График</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>190</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>190</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_xmin"/>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lineEdit_func"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>y, max</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_xmax"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_ymin"/>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_plot">
       <property name="text">
        <string>Построить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>x, min</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>y, min</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_ymax"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Функция:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>x, max</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Область значений:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Область определения:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>libs/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
